### DISCUSSION TITLE
Add support for tabs in markdown documentation

### DISCUSSION THREAD

**Discussion by arbh89** [scalar/scalar#3034] [Ideas]
It would be nice to have the feature to support tabs in markdown documentation.

Something like this

![image](https://github.com/user-attachments/assets/2d294f0d-f62d-405f-ac2c-57ca0e5c96ed)

**Comment by hanspagel** [comment]
Great suggestion! We’re looking into support this, it’ll take time, though. It’s not as straight-forward as it might seem. :v:
